### PR TITLE
Introduce Appfs telemetry

### PR DESCRIPTION
**Pull request by SiskaPavel**

### CODE REVIEW

File: ipfixprobe.spec.in [hunk 1]
```diff
@@ -57,8 +57,8 @@ Vendor: CESNET, z.s.p.o.
 Packager: @USERNAME@ <@USERMAIL@>
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}
 Summary: IPFIX flow exporter with various extending IPFIX elements exported by plugins.
-Requires: libatomic
-BuildRequires: gcc gcc-c++ make doxygen pkgconfig libatomic
+Requires: libatomic fuse3
+BuildRequires: gcc gcc-c++ make doxygen pkgconfig libatomic telemetry
```
Comment (cejkato2) [inline ipfixprobe.spec.in:61]: This should be checked again - BuildRequires has no fuse3-devel or something like that; Requires has no telemetry - is there anything needed?

Comment (cejkato2) [reply ipfixprobe.spec.in:61]: For ubuntu there is libfuse3-dev installed...

File: input/dpdk.cpp [hunk 2]
```diff
@@ -239,19 +275,24 @@ InputPlugin::Result DpdkReader::get(PacketBlock& packets)
...
-        m_parsed++;
 #endif
     }
 
+    m_seen += recivedPackets;
+    m_parsed += recivedPackets;
+
+    m_stats.receivedPackets += recivedPackets;
+    m_stats.receivedBytes += packets.bytes;
+
```
[the remark this answers is not in the document]
Comment (SiskaPavel) [reply input/dpdk.cpp]: fixed

File: ipfixprobe.spec.in (hunk 1 above)
Comment (SiskaPavel) [reply ipfixprobe.spec.in]: fixed

**Review by cejkato2 — APPROVED** [review]

**Pull request merged**